src/utilities/utility.ts

```diff
@@ -74,16 +74,23 @@ export function annotateChildPods(kubectl: Kubectl, resourceType: string, resour
 }
 
 export function annotateNamespace(kubectl: Kubectl, namespaceName: string): IExecSyncResult {
-    let annotate = true;
     const result = kubectl.getResource('namespace', namespaceName);
-    this.checkForErrors([result]);
-    const annotationsSet = JSON.parse(result.stdout).metadata.annotations;
-    if (!!annotationsSet && !!annotationsSet.runUri && annotationsSet.runUri.indexOf(process.env['GITHUB_REPOSITORY']) == -1) {
-        annotate = false;
-        core.debug(`Skipping 'annotate namespace' as namespace annotated by other workflow`);
+    if (result == null) {
+        return <IExecSyncResult>{ code: -1, stderr: 'Failed to get resource' };
     }
-    if (annotate) {
-        return kubectl.annotate('namespace', namespaceName, workflowAnnotations, true);
+    else if (!!result && !!result.stderr) {

```

Comment:
`!!` is redundant here right?

Reply:
I checked for `!!` in conditions for JS. It ensures the resulting type is a boolean. [Ref](https://stackoverflow.com/questions/7452720/what-does-the-double-exclamation-operator-mean)

Reply:
It does.. I agree..  but it's also redundant.. adds to value in this case

Reply:
This `!!` thing is inconsistent across the file. So I suggested to remove it

Reply:
Will change it.